great_expectations/analytics/anonymizer.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import annotations
 
-from hashlib import md5
+from hashlib import sha256
 
 
 def anonymize(string: str) -> str:
-    return md5(string.encode("utf-8")).hexdigest()
+    return sha256(string.encode("utf-8")).hexdigest()
```

Comment:
This is used for sending analytic events and we would care if the output of this method changed. I would either:
1. add a setting so this could be parameterized and default to md5.
2. try md5 first and if it fails fallback to sha256.